__tests__/app.test.js

```diff
@@ -115,3 +115,54 @@ describe("GET /api/users", () => {
       });
   });
 });
+
+describe.only("PATCH /api/articles/:article_id", () => {
```

Comment:
You must remove the `.only` and make sure that all of the tests are passing before proceeding